settings.php

```diff
@@ -56,4 +56,8 @@
             new tool_cloudmetrics\admin_setting_manage_metrics()
         );
     }
+
+    foreach (cltr::get_enabled_plugins() as $plugin) {
```

Comment:
It is often useful to be able to configure plugins before they are enabled like auth plugins do

Reply:
Done